include/roboteam_utils/Vector2.h

```diff
@@ -6,7 +6,7 @@
 
 namespace rtt {
 
-    class Angle;
+class Angle;
```

Comment:
this has no use, you cannot have an lvalue reference to an incomplete type so you have to include it anyway.

The only thing you can have to an incomplete type, is a pointer, hence why void* is allowed but void& is not

Reply:
WDYM? I wrote a test and it passes. As long as Angle.h is included for generating the code for the definition of the function, there's no problem with the lvalue reference being undefined here.
The only thing it does do is force users to explicitly include Angle.h if they want to use this constructor in their files.

Reply:
https://stackoverflow.com/questions/7356603/references-to-incomplete-types. Don't know how relevant this still is since it's old, but if it was allowed 8 years back it should still be so now.

Reply:
my bad did not see the tests.

Also, my bad, you're right, having an lvalue reference to an incomplete type is fine, not sure why that's stuck in my head.